projects/plugins/jetpack/extensions/blocks/story/story.php

```diff
@@ -186,13 +212,20 @@ function render_video( $media ) {
 	}
 
 	if ( ! empty( $poster_url ) ) {
+		$poster_width  = esc_attr( $meta_width );
+		$poster_height = esc_attr( $meta_height );
+		$content_width = intval( Jetpack::get_content_width() );
+		if ( is_numeric( $content_width ) ) {
+			$poster_height = round( ( $content_width * $poster_height ) / $poster_width );
+			$poster_width  = $content_width;
+		}
 		return sprintf(
 			'<img title="%1$s" alt="%2$s" class="%3$s" src="%4$s"%5$s%6$s>',
 			esc_attr( get_the_title( $media['id'] ) ),
 			esc_attr( $description ),
 			'wp-block-jetpack-story_image wp-story-image ' .
 			get_image_crop_class( $meta_width, $meta_height ),
-			esc_attr( $poster_url ),
+			add_query_arg( 'resize', rawurlencode( $poster_width . ',' . $poster_height ), esc_attr( $poster_url ) ),
```

Comment:
I would suggest escaping last:

```suggestion
			esc_attr( add_query_arg( 'resize', rawurlencode( $poster_width . ',' . $poster_height ), $poster_url ) ),
```